plugin/fireplace.vim

```diff
@@ -1669,41 +1681,33 @@ augroup END
 " Section: Spec
 " TODO: `spec-list`
```

Comment:
What's the story here? I would imagine that switching from maps to commands would make this one easy to implement?

Reply:
Removed that comment.  It's basically `(spec/registry)` and I assume, that in emacs/cider one opens that list and then navgiates to form/example from there.  AFAIK there currently is nothing similar in fireplace to add this to.